app/src/main/res/values/strings.xml

```diff
@@ -9,6 +9,10 @@
     <string name="message_cancel">CANCEL</string>
     <string name="message_delete">DELETE</string>
     <string name="message_unknown_exception">Unknown exception</string>
+
+    <!--Battery Optimization-->
+    <string name="battery_optimization_turn_off_message">We have to turn off battery optimization for correct work</string>
```

Comment:
It sounds like a good explanation for a developer, but we need to rephrase it for a user.
I'll get back soon with some suggestions

Reply:
Please change text to “Do you agree to disable battery optimization for correct work of timer?”